src/database/database.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+
+import { MsSqlCedarDatabaseModule } from './mssql-cedar-database.module';
+import { MsSqlCisDatabaseModule } from './mssql-cis-database.module';
+import { MsSqlMdmDatabaseModule } from './mssql-mdm-database.module';
+import { MsSqlNumberGeneratorDatabaseModule } from './mssql-number-generator-database.module';
+
+@Module({
+  imports: [MsSqlMdmDatabaseModule, MsSqlCedarDatabaseModule, MsSqlCisDatabaseModule, MsSqlNumberGeneratorDatabaseModule],
+  exports: [MsSqlMdmDatabaseModule, MsSqlCedarDatabaseModule, MsSqlCisDatabaseModule, MsSqlNumberGeneratorDatabaseModule],
+})
+export class DatabaseModule {}
```

Comment:
Now that we have a `DatabaseModule`, is there any reason why the `database` folder isn't under `module`?

Reply:
this is updated now